puppet/environments/box/modules/profiles/manifests/box/cockpit.pp

```diff
@@ -0,0 +1,36 @@
+class profiles::box::cockpit {
+
+  package { 'cockpit': }
+
+  ini_setting { 'Cockpit LoginTitle':
+    ensure    => present,
+    path      => '/etc/cockpit/cockpit.conf',
+    section   => 'WebService',
+    setting   => 'LoginTitle',
+    value     => $::fqdn,
+    show_diff => true,
+  }
+
+  ini_setting { 'Cockpit MaxStartups':
+    ensure    => present,
+    path      => '/etc/cockpit/cockpit.conf',
+    section   => 'WebService',
+    setting   => 'MaxStartups',
+    value     => 10,
+    show_diff => true,
+  }
+
+  ini_setting { 'Cockpit AllowUnencrypted':
```

Comment:
@yiendos When making a change to the configuration of an application, you want to make sure the service gets restarted. You need to use the `notify` parameter for this. Eg. `notify => Service['cockpit']`. This will have the service restart so the change will be picked up.

Reply:
@yiendos Update, the `notify` parameter is required on all three ini_settings!